blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -123,9 +123,14 @@ public static void track(HttpRequest request) {
         }
     }
 
-    public static void trackDelayedMetricsTenant(String tenantid) {
+    public static void trackDelayedMetricsTenant(String tenantid, List<String> metricNames) {
         if (isTrackingDelayedMetrics) {
-            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metric is %s",tenantid);
+            StringBuilder sb = new StringBuilder();
+            for(String name : metricNames) {
+                sb.append(name);
+                sb.append(";");
+            }
+            String logMessage = String.format("[TRACKER][DELAYED METRIC] Tenant sending delayed metric is %s with the delayed metrics -- %s",tenantid,sb.toString());
```

Comment:
this is hard to do during rollup.  How about we add another tracker interface for the Requests handler, that will allow us to turn on tracking for tenantID AND metric name?  Once we have a few of these tenant ids + metricnames coming in that are flagged as delayed metrics, we can turn on the tracking at the request end and see the exact ingestion time and entire payload.